src/features/featurecreator.sh

```diff
@@ -30,14 +30,11 @@ cp -r _featureTemplate/ $feature/
 
 cd $feature/
 
-sed -i -deleteme "s/blank/$feature/g" container.js reducer.js components/Blank.js
-sed -i -deleteme "s/BLANK/$FEATURE/g" actionTypes.js actions.js reducer.js
-sed -i -deleteme "s/Blank/$Feature/g" container.js initialState.js reducer.js components/Blank.js
-mv components/Blank.js components/$Feature.js
-mv components/Blank.scss components/$Feature.scss
-mv components/BlankTextInput.js components/${Feature}TextInput.js
-
-rm *-deleteme components/*-deleteme
+#sed -i -deleteme "s/blank/$feature/g" container.js reducer.js components/component.js
```

Comment:
We can just delete this shell script, the template is designed to be cp'd now. We can add simple automations to add it to the reducer down the road.